internal/dataplane/sendconfig/kong.go

```diff
@@ -13,14 +19,69 @@ import (
 
 // Kong Represents a Kong client and connection information.
 type Kong struct {
-	URL string
-	// If the gateway instance does not support tags, pass an empty FilterTags slice instead.
-	FilterTags []string
+	Clients []ClientWithPluginStore
+
 	// Headers are injected into every request to Kong's Admin API
 	// to help with authorization/authentication.
-	Client            *kong.Client
-	PluginSchemaStore *util.PluginSchemaStore
-	InMemory          bool
-	Version           semver.Version
-	Concurrency       int
+	InMemory bool
+	// Currently, this assumes that all underlying clients are using the same version
+	// hence this shared field in here.
+	Version     semver.Version
```

Comment:
For now (the initial patch that introduces the concept of configuring multiple gateways at once) I'd prefer to leave this in as is.

This will go through a several iterations of refactor to factor in: multiple versions (upgrades) and dynamically removing and adding instances in runtime (not only on startup).